core/domain/suggestion_registry.py

```diff
@@ -1481,6 +1481,205 @@ def to_dict(self):
         }
 
 
+class TranslationReviewStats:
+    """Domain object for the TranslationReviewStatsModel."""
+
+    def __init__(
+            self, language_code, contributor_user_id, topic_id,
+            reviewed_translations_count, reviewed_translation_word_count,
+            accepted_translations_count,
+            accepted_translations_with_reviewer_edits_count,
+            accepted_translations_with_reviewer_edits_word_count,
+            first_contribution_date, last_contribution_date):
+        self.language_code = language_code
+        self.contributor_user_id = contributor_user_id
+        self.topic_id = topic_id
+        self.reviewed_translations_count = reviewed_translations_count
+        self.reviewed_translation_word_count = reviewed_translation_word_count
+        self.accepted_translations_count = accepted_translations_count
```

Comment:
[In my final mocks for translation review stats](https://www.figma.com/file/SNOHAatFcCvdvIAdxAzF6E/Contributor-Dashboard-Achievements?node-id=112%3A16), there is no field for `accepted_translations_with_reviewer_edits_word_count` so this probably can be removed. I added it here only because this field was included in the storage model. So this field can be removed from both the domain objects and storage models.